script/main.py

```diff
@@ -282,7 +282,31 @@ def users_born(conn, start_date, end_date):
     full_date_of_birth = datetime.strptime(user_date_birth, '%Y-%m-%dT%H:%M:%S.%fZ')
     date_of_birth = full_date_of_birth.date()
     print(f'User {user_name} {user_last_name} was born in {date_of_birth}')
-  
+
+
+def most_secure_password(conn):
+  passwords_list =  [i[2] for i in select_all_gender_and_dob_date_and_password(conn)]  
```

Comment:
Better select records here instead of calling external function.
Usually when you have to change function every time you add new functionality, it means that there is something wrong with this function